### PR TITLE
Docs touchup

### PR DESCRIPTION
**Pull request by AfoHT**
Unleashed some language linters on the book

### CODE REVIEW

File: book/en/src/awesome_rtic.md [hunk 1]
```diff
@@ -1 +1,8 @@
 # Awesome RTIC examples
+
+See the [`rtic-rs/rtic-examples`][rticexamples] repository for community
+provided complete examples.
+
+Pull-requests to this repo is welcome!
```
Comment (korken89) [inline book/en/src/awesome_rtic.md:6]: are welcome

**Review by korken89 — CHANGES_REQUESTED** [review]
Started reviewing, not done yet

**Review by perlindgren — APPROVED** [review]
Looks good, I made a few changes directly in the git web (never tried that before), let's see how it pans out. Besides these small changes, I think the docs are good to go.

**Review by korken89 — APPROVED** [review]

**Comment by korken89** [comment]
bors merge

**Pull request merged**